### ISSUE TITLE
react-datetime going out of screen

### ISSUE THREAD

**Issue by trybotics** [YouCanBookMe/react-datetime#356]
react-datetime going out of screen
![screenshot from 2017-06-23 13-20-42](https://user-images.githubusercontent.com/22857102/27471838-cfd3de12-5816-11e7-9acc-e6984875e868.png)

**Comment by simeg** [comment]
Hi @trybotics.

How do you think the component should act in this scenario?

**Comment by jsobus-neurosys** [comment]
@simeg I think it should open with alignement to the right end of input.

**Comment by simeg** [comment]
@jsobus-neurosys Any idea on how to implement your proposed solution? Maybe there could be a prop that would decide if it would render on left or right side of the input field, but I don't know how to create a single general solution that would adapt to the screen width.

**Comment by jsobus-neurosys** [comment]
Yes a prop that decide if it would render on left or right is a good begining :) It can be also handled automatically with some kind of collision detection of the screen size. I found jQuery solution (which we don't like to add) which You can analyze and implement inside component https://css-tricks.com/collision-detection/. There is also thing called http://tether.io/ which is wrapped with https://github.com/souporserious/react-tether that may be used for that problem.

**Comment by simeg** [comment]
Looks interesting, I didn't know about this :) I'll dig deeper into this later. Thanks.

**Comment by uwolfer** [comment]
In my case, following CSS helped:
```css
.rdtPicker {
  right: 0;
}
```

**Comment by rurihiiragi** [comment]
Would it be possible to have a renderDatetimePicker option similar to renderInput? Running into a similar issue and would like to render the datetime picker as a modal that's centered on the screen.

**Comment by simeg** [comment]
@rurihiiragi You could wrap this component in a modal component, no need for an extra prop.

**Comment by rurihiiragi** [comment]
You mean the <Datetime /> component? I only want the datetime picker to be in a model, but the <input> remains as part of a form.

**Comment by simeg** [comment]
Oh, that makes it a bit more complicated. You want to decouple them visually. That is not possible at the moment using props, but maybe with a bit of CSS magic? Not sure.

**Comment by saulojoab** [comment]
Does anyone know how to center the Datetime on the input? For example, mine is on the left side:
![image](https://user-images.githubusercontent.com/37988252/57112175-4754ee80-6d15-11e9-9a50-d6fb9fb65eb9.png)
I can make it go to the right side by using:
```css
.rdtPicker {
  right: 0;
}
```
But I have no idea how to center it. Maybe I'm just dumb :p
